src/components/cc-ssh-key-list/cc-ssh-key-list.js

```diff
@@ -141,6 +149,33 @@ export class CcSshKeyList extends LitElement {
     }
   }
 
+  _isPublicKeyPrivate (publicKey) {
+    const lowerCasePublicKey = publicKey.toLowerCase();
+    return lowerCasePublicKey.match(' private ') || lowerCasePublicKey === 'private';
+  }
+
+  _onNameInput ({ detail: name }) {
+    // ensuring model is synchronised with the view
+    this.createSshKeyForm = {
+      ...this.createSshKeyForm,
+      name: {
+        ...this.createSshKeyForm.name,
+        value: name,
+      },
+    };
+  }
+
+  _onPublicKeyInput ({ detail: publicKey }) {
+    // ensuring model is synchronised with the view
+    this.createSshKeyForm = {
+      ...this.createSshKeyForm,
+      publicKey: {
+        ...this.createSshKeyForm.publicKey,
+        value: publicKey,
+      },
+    };
+  }
```

Comment:
I think I understand why we need to go from uncontrolled form inputs to controlled but I wonder if a `live` could be enough?

I also know we shoot ourselves in the foot with `live` a few times :disappointed:

Reply:
I'll give it a try!